examples/infinity_feedliker_example.py

```diff
@@ -0,0 +1,21 @@
+"""
+    instabot example
+
+    Workflow:
+        Like rescent medias from your timeline feed.
+"""
+
+import sys
+import os
+import time 
+
+sys.path.append(os.path.join(sys.path[0], '../'))
+
+from instabot import Bot
+
+bot = Bot()
+bot.login()
+
+while True:
+	bot.like_timeline()
+	time.sleep(1*60)
```

Comment:
@maxkrivich if 2 or more mins its okay to use 2*60 or something like this.
Also, if you want to use min, use variable for this, for example

```python
wait = 2*60 # Minutes to seconds
time.sleep(wait)
```
Make sense?